templates/client-auth-n.cfg.xml.erb

```diff
@@ -3,7 +3,7 @@
 <client-auth xmlns="http://<%= scope.lookupvar('::repose::cfg_namespace_host') %>/repose/client-auth/v1.0">
 <%- if ! @delegating.nil? -%>
   <%- if @delegating %>
-    <delegating/>
+    <delegating<%- if ! @delegating_quality.nil? -%> quality="<%= @delegating_quality %>"<%- end -%>/>
```

Comment:
Careful with the -%> as it'll nuke that space before quality.  In this case, it's best not to put the - here